cfg/cis-1.6/master.yaml

```diff
@@ -118,8 +118,16 @@ groups:
 
       - id: 1.1.9
         text: "Ensure that the Container Network Interface file permissions are set to 644 or more restrictive (Manual)"
```

Comment:
` (Manual)` should be removed now too, right? I can write a PR to fix that if accurate

Reply:
No, because the Manual/Automated is from CIS, it is what used to be scored/not scored